src/extract.ts

```diff
@@ -304,6 +304,69 @@ function extractGoogleCppResult(output: string): BenchmarkResult[] {
     });
 }
 
+function extractCatch2Result(output: string): BenchmarkResult[] {
+    const lines = output.split('\n');
+
+    const ret = [];
+    // Example:
+    //   test bench_fib_20 ... bench:      37,174 ns/iter (+/- 7,527)
+    const reTestCaseStart = /^(benchmark name) +(samples) +(iterations) +(estimated)/;
+    const reBenchmarkStart = /^([a-zA-Z\d ]+) +([\d]+) +([\d]+) +([\d]+\.[\d]+) (ms|us)/;
+    const reBenchmarkValues = /^ +([\d]+\.[\d]+) (us|ms) +([\d]+\.[\d]+) (us|ms) +([\d]+\.[\d]+) (us|ms)/;
```

Comment:
Let me ask three things here:

- What is the reason to use `[\d]` rather than `\d`? If there is no special reason, please prefer simpler one.
- It always expects `.` in numbers. Is there no case where it does not include `.` like integer value `1234567`?
- Only `us` and `ms` are considered. Doesn't `s` appear for 'seconds'?

Reply:
the `[ ]` brackets are indeed not needed. Will remove them. 
Will add the support for non-decimal integers. I've never seen it so far, but it might happen
As far as I know catch2 will only produce us and ms values, but I cannot guarantee it. Will add seconds to increase robustness.

Reply:
Unfortunately adding proper decimal parsing support made the regex a bit more complicated, but I could implement the rest of your suggestions